site/content/docs/developing/package-addons.en.md

```diff
@@ -0,0 +1,155 @@
+# Package Addons <span class="version" > added in [v1.21.0](../../blogs/release-v121.en.md) </span>
```

Comment:
I understand if it's deemed too big and we want a separate page for this, in that scenario should we create a new section under developing like this:
```
    - Developing:
      - Additional AWS Resources:
          - Modeling With AWS CloudFormation: docs/developing/addons/modeling.en.md
          - Packaging Local Artifacts: docs/developing/addons/package.en.md
```
We would need to add a redirect as well so that the old URL reroutes to the new hierarchy:
```
    - redirects:
        redirect_maps:
            'docs/developing/additional-aws-resources.en.md': 'docs/developing/addons/modeling.en.md'
```